pyocd/probe/tcp_probe_server.py

```diff
@@ -410,6 +411,9 @@ def _request__get_memory_interface_for_ap(self, ap_address_version, ap_nominal_a
             handle = None
         return handle
     
+    def _request__swo_read(self):
+        return list(self._probe.swo_read())
```

Comment:
Should the client convert the list back into a bytearray? It's not technically necessary,  but might be nice for symmetry.

Reply:
I was trying to keep with what appeared to be the intended signature in the comments of returning a list of integers.